### PR TITLE
Pn 9292

### PR DESCRIPTION
**Pull request by azhansultan**
* Added new feature files related to Mittente send notification.

### CODE REVIEW

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/LoginPersonaFisicaPagoPA.java [hunk 1]
```diff
@@ -89,39 +89,6 @@ public void loginMittenteConTokenExchange(String personaFisica) {
         notifichePFPage.waitLoadNotificheDEPage();
     }
 
-    @Given("PF - Si effettua la login tramite token exchange con utente {string} e viene visualizzata la dashboard")
```
Comment (leleOFA) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/LoginPersonaFisicaPagoPA.java:92]: why delete this method?

**Comment by leleOFA** [comment]
wait fix creation with api for notification

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/LoginPersonaFisicaPagoPA.java (hunk 1 above)
Comment (azhansultan) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/LoginPersonaFisicaPagoPA.java:92]: not used

File: src/test/resources/feature/2-mittente/1_invioNotifiche/invioNotificaMonoPFNonValido.feature [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    Then Nella section Informazioni preliminari si inseriscono i dati della notifica
+      | oggettoNotifica   | Pagamento rata IMU |
+      | descrizione       | PAGAMENTO RATA IMU |
+      | gruppo            | test-TA-FE-TEST    |
+      | codiceTassonomico | 123456A            |
+      | modalitaInvio     | A/R                |
+    And Cliccare su continua
+    And Si visualizza correttamente la pagina Piattaforma Notifiche section Destinatario
+    Then Nella section Destinatario si inseriscono i dati del destinatario
+      | soggettoGiuridico       | PF                |
```
Comment (leleOFA) [inline src/test/resources/feature/2-mittente/1_invioNotifiche/invioNotificaMonoPFNonValido.feature:21]: here create notification with rest is not implemented

File: src/test/resources/feature/2-mittente/1_invioNotifiche/invioNotificaMonoPFNormalizzazioneKOLovelace.feature [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    Then Nella section Informazioni preliminari si inseriscono i dati della notifica
+      | oggettoNotifica   | Pagamento rata IMU |
+      | descrizione       | PAGAMENTO RATA IMU |
+      | gruppo            | test-TA-FE-TEST    |
+      | codiceTassonomico | 123456A            |
+      | modalitaInvio     | A/R                |
+    And Cliccare su continua
+    And Si visualizza correttamente la pagina Piattaforma Notifiche section Destinatario
+    Then Nella section Destinatario si inseriscono i dati del destinatario
+      | soggettoGiuridico       | PF               |
```
Comment (leleOFA) [inline src/test/resources/feature/2-mittente/1_invioNotifiche/invioNotificaMonoPFNormalizzazioneKOLovelace.feature:21]: as before

File: src/test/resources/feature/2-mittente/1_invioNotifiche/invioNotificaMonoPGDomicilioDigitaleKO.feature [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    Then Nella section Informazioni preliminari si inseriscono i dati della notifica
+      | oggettoNotifica   | Pagamento rata IMU |
+      | descrizione       | PAGAMENTO RATA IMU |
+      | gruppo            | test-TA-FE-TEST    |
+      | codiceTassonomico | 123456A            |
+      | modalitaInvio     | A/R                |
+    And Cliccare su continua
+    And Si visualizza correttamente la pagina Piattaforma Notifiche section Destinatario
+    Then Nella section Destinatario si inseriscono i dati del destinatario
+      | soggettoGiuridico       | PG          |
```
Comment (leleOFA) [inline src/test/resources/feature/2-mittente/1_invioNotifiche/invioNotificaMonoPGDomicilioDigitaleKO.feature:21]: as before

File: src/test/resources/feature/2-mittente/1_invioNotifiche/invioNotificaMonoPGDomicilioDigitaleOK.feature [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    And Nella pagina Piattaforma Notifiche cliccare sul bottone Invia una nuova notifica
+    And Si visualizza correttamente la pagina Piattaforma Notifiche section Informazioni preliminari
+    Then Nella section Informazioni preliminari si inseriscono i dati della notifica
+      | oggettoNotifica   | Pagamento rata IMU |
+      | descrizione       | PAGAMENTO RATA IMU |
+      | gruppo            | test-TA-FE-TEST    |
+      | codiceTassonomico | 123456A            |
+      | modalitaInvio     | A/R                |
+    And Cliccare su continua
+    And Si visualizza correttamente la pagina Piattaforma Notifiche section Destinatario
```
Comment (leleOFA) [inline src/test/resources/feature/2-mittente/1_invioNotifiche/invioNotificaMonoPGDomicilioDigitaleOK.feature:16]: as before

**Review by leleOFA — APPROVED** [review]

**Pull request merged**